GOVERNANCE.md

```diff
@@ -166,19 +166,18 @@ Provide a summary of the nominee's contributions. For example:
   organization
 * Other participation in the wider Node.js community
 
-Mention @nodejs/collaborators in the issue to notify other collaborators about
-the nomination.
+If the private discussion receives some support (in form of explicit comments or
+thumbs-up reactions) and no explicit block after a few days, add a comment in the
+private discussion stating you're planning on opening a public issue, e.g.
+"I see a number of approvals and no block, I'll be opening a public
+nomination issue if I don't hear any objections in the next 72 hours".
+After sufficient time has passed, open an issue in the [nodejs/node][] repository.
+Provide the same summary of the nominee's contributions.
 
 The nomination passes if no collaborators oppose it after one week. In the case
```

Comment:
```suggestion
 The nomination passes if no collaborators explicitly oppose the public nomination after one week. In the case
```

I would suggest we also include a definition of what explicit opposition means. Simply asking a clarifying question could be viewed as "opposition" by some and not by others.

Reply:
I've decided to not try to define what a block is, I think the informal thing is enough given it's an optional step anyway. I'd expect the "I'm going to open the public issue in 72 hours" be enough to let folks who are actually blocking to react in time in case their blocking comment wasn't explicit enough